edge/edge.go

```diff
@@ -104,16 +99,11 @@ func (manager *Manager) startEdgeBackgroundProcessOnDocker(runtimeCheckFrequency
 		return err
 	}
 
-	ticker := time.NewTicker(runtimeCheckFrequency)
-
 	go func() {
-		for {
-			select {
-			case <-ticker.C:
-				err := manager.checkDockerRuntimeConfig()
-				if err != nil {
-					log.Printf("[ERROR] [internal,edge,runtime,docker] [message: an error occurred during Docker runtime configuration check] [error: %s]", err)
-				}
+		ticker := time.NewTicker(runtimeCheckFrequency)
+		for range ticker.C {
+			if err := manager.checkDockerRuntimeConfig(); err != nil {
```

Comment:
for consistency i'd keep the following pattern (which I find more readable)

```
err := instruction()
if err != nil {
...
}
```

VS

```
if err := instruction(); err != nil {
...
}
```